docker/scripts/java-15-vars.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+JAVA_INSTALL_VERSION_MAJOR='15'
+JAVA_INSTALL_VERSION=2
+JAVA_INSTALL_PATH=/usr/opt/java-${JAVA_INSTALL_VERSION_MAJOR}-${JAVA_INSTALL_VERSION}/
+JDK_FILE=/tmp/jdk-15.0.2_linux-x64_bin.tar.gz
```

Comment:
This version doesn't exist, does it?

Reply:
That's the pattern there for quite some time. Not sure what it means. Do we have to ask someone e.g. @abal5 to install that for us ? Is this mounted ?

Reply:
It doesn't, adding.

Reply:
Added - https://s3.console.aws.amazon.com/s3/buckets/artifacts.h2o.ai?region=us-east-1&prefix=releases/oracle/jdk-15/
Commit: https://github.com/h2oai/h2o-3/pull/5110/commits/350a230a7a6cfe2366d34d761c4f9bc8acb059fb
Wanted to name it in a better way, but the shell scripts kind of get in the way, so  I kept the current naming pattern.